app/models/timer.rb

```diff
@@ -0,0 +1,48 @@
+class Timer < ActiveRecord::Base
+
+  belongs_to :player
+
+  # ALTERNATIVE start and stop methods
+  
+  # def start_time
+  #   @running = true
+  #   @start_time = Time.now
+
+  #   limit if @time_limit
+
+  #   self
+  # end
+
+  # def stop_time
+  #   if @running
+  #     defuse
+  #     @end_time = Time.now
+  #     @running = false
+  #     @total_time += (@end_time - @start_time)
+  #   else
+  #     nil
+  #   end
+  # end
+
+
+  def start_time
+    Timer.create(player_id: current_player.id, time: Time.now)
```

Comment:
This is creating a new timer each start, is that the intent?

This could be useful from a historical logging perspective, but the reset method kind of conflicts with that intent. If you reset, then start, all the timers in history are zero.